Realm/RLMDecimal128.h

```diff
@@ -49,9 +49,9 @@ NS_ASSUME_NONNULL_BEGIN
 
 /// Parses the given string to a RLMDecimal128.
 ///
-/// Returns `nil` and sets `error` if the string cannot be parsed as a
-/// RLMDecimal128.
-- (nullable instancetype)initWithString:(NSString *)string error:(NSError *_Nullable*)error;
```

Comment:
The old documentation here was incorrect; we actually only returned `nil` and set `error` for syntactically valid numbers which were outside the range of what the old parsing code supported, and we already returned nan for invalid syntax.